Italian/antonio_conte/antonio.md

```diff
@@ -0,0 +1,9 @@
+TELEFONATA AD ANTONIO CONTE (A.C.)
+
+Io: We, Mister! Come andiamo? Mi sono svegliato in una [stanza molto strana](http://giphy.com/gifs/uy4zM12rn3oaY). Che cosa devo fare?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.